packages/gestalt/src/Tag.css

```diff
@@ -1,24 +1,71 @@
 .button {
   composes: noBorder from "./Borders.css";
-  composes: paddingX2 from "./boxWhitespace.css";
-  composes: paddingY3 from "./boxWhitespace.css";
   composes: pointer from "./Cursor.css";
+
+  /*
+  Since the button is on an already transparent bg, it appears darker. This adjusts the background color to be a tad brighter since it's layered on top.
+  */
+  backdrop-filter: brightness(1.1);
```

Comment:
@rlingineni it feels hacky to me. Would changing the colors instead work in this case? 🤔

Reply:
Agree, it is a little hacky. It was something I figured out w/ TagData.

https://github.com/pinterest/gestalt/blob/646c4107ec3283f449001574f14a03d800bb8df9/packages/gestalt/src/TagData.css#L36

Since we're overlaying the buttons, and bg color can be dynamic, it uses this

Reply:
Actually, I removed it, and it seemed to work. I got rid of it. Good catch. I kept it for warning base to retain a darker bg